security-release-process.md

```diff
@@ -0,0 +1,597 @@
+- The Fix Lead will remove the Fix Team from the private security repo once it is no longer needed.
+
+Communications process:
+- The [Private Distributors List](#private-distributors-list) will be notified at least 24 hours in
+  advance of a pending release containing security vulnerability fixes with the public messaging,
+  date, and time of the announcement.
+- The Fix Lead will announce the new releases, the CVE number, severity, and impact, and the
+  location of the binaries to get wide distribution and user action. As much as possible this
+  announcement should be actionable, and include any mitigating steps users can take prior to
+  upgrading to a fixed version. The recommended target time is 4pm UTC on a non-Friday weekday. This
+  means the announcement will be seen morning Pacific, early evening Europe, and late evening Asia.
+  The announcement will be sent via the following channels:
+  - General announcement email ([template](comms-templates/vulnerability-announcement-email.md)) to
+    multiple Kubernetes lists
+  - OSS-Security announcement email
+    ([template](comms-templates/vulnerability-announcement-email.md)) to
+    `oss-security@lists.openwall.com`
+  - `#announcements` slack channel ([template](comms-templates/vulnerability-announcement-slack.md))
+  - [discuss.kubernetes.io](https://discuss.kubernetes.io/c/announcements) forum (this should be
+    posted automatically using the general announcement email template)
+  - Tracking issue opened in https://github.com/kubernetes/kubernetes/issues
+    ([template](comms-templates/vulnerability-announcement-issue.md)) and prefixed with the
+    associated CVE ID (if applicable)
+  - Medium and Low severity vulnerability fixes that will be released as part of the next Kubernetes
+    [patch release](https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md)
+    will have the fix details included in the patch release notes. Any public announcement sent for
+    these fixes will link to the release notes.
+
+## Private Distributors List
+
+This list is used to provide actionable information to multiple distribution
+vendors at once.
+
+See the [private distributor list doc](private-distributors-list.md) for more information.
+
+## Retrospective
+
+These steps should be completed 1-3 days after the Release Date. The retrospective process [should be blameless](https://landing.google.com/sre/book/chapters/postmortem-culture.html).
+
+- The Fix Lead will send a retrospective of the process to kubernetes-dev@googlegroups.com including details on everyone involved, the timeline of the process, links to relevant PRs that introduced the issue, if relevant, and any critiques of the response and release process.
+- The Release Managers and Fix Team are also encouraged to send their own feedback on the process to kubernetes-dev@googlegroups.com. Honest critique is the only way we are going to get good at this as a community.
```

Comment:
We haven't been following this process. I still think it would be good to do this. Maybe as a presentation to sig-security.